src/index.js

```diff
@@ -1,13 +1,30 @@
 const express = require('express');
 const config = require('config');
 const app = express();
+app.use(express.json());
+
+let response = {};
+
 
 const { port: PORT, host, prefix } = config.get('api');
 
 app.get(`${prefix}/account`, (req, res) => {
-  res.send();
+  res.send(response);
 });
 
+app.put(`${prefix}/account`, (req, res) => {
+const {body} = req;
+response = body;
+
+res.send(response);
```

Comment:
Could you please respect indentation inside your callback like in your `app.delete` ?